python/sim/containers.w

```diff
@@ -170,7 +172,11 @@ pub class Container {
 
       if let port = opts.port {
         dockerRun.push("-p");
-        dockerRun.push("{port}");
+        if let exposedPort = opts.exposedPort {
+          dockerRun.push("{exposedPort}:{port}");
+        } else {
+          dockerRun.push("{port}");
+        }
```

Comment:
@eladcon EladB has reworked the sim.Container recently, I wonder if it's usable for this winglib now